server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -181,6 +180,20 @@ private static boolean isHtml(IndexableMessage im) {
             && im.getSubType().equals(MediaType.HTML_UTF_8.subtype());
     }
 
+    private static List<Attachment> getAttachments(List<org.apache.james.mailbox.store.mail.model.Attachment> attachments) {
+        return attachments.stream()
+                .map(Message::fromAttachment)
+                .collect(org.apache.james.util.streams.Collectors.toImmutableList());
```

Comment:
It would be cool to rename our collectors to ImmutableCollectors to avoid such name conflicts